### ISSUE TITLE
Stocks API Heroku app seems to be down

### ISSUE THREAD

**Issue by ream720** [angular-in-action/stocks#7]
Hi, I think the stocks API hosted on Heroku may be down or no longer accessible since Heroku did away with free hosting.

Is there any chance you could check into this? I'd really appreciate it. 
![stocksapiherokudown](https://user-images.githubusercontent.com/24925392/198896999-41a2dd2f-be19-4557-b301-7df8ffea8b6f.JPG)

**Comment by gnomeontherun** [comment]
Yes I saw this and have been working on an update. The Heroku change is frustrating as it will break all of the books, but it looks like it should work again on Nov 1 until the actual shutdown. 

I'm going to change the service enough so people can run it locally, and avoid the problems of a remote server.

**Comment by ream720** [comment]
@gnomeontherun thanks for your continued dedication! This book is a great resource.

**Comment by gnomeontherun** [comment]
I've updated this project and the API service so it can work without Heroku. Look at https://github.com/angular-in-action/api and the `main` branch on this repository to get the updates.
